documentation/graph-analytics.adoc

```diff
@@ -1,3 +1,158 @@
-== Setup Instructions
+== Introduction
 
-See https://neo4j.com/docs/aura/graph-analytics/[documentation]
+link:https://neo4j.com/docs/aura/graph-analytics[Graph Analytics^] allows you to run graph algorithms on your data, for example to detect communities or shortest paths between nodes.
+
+== Requirements
+
+You must create link:https://neo4j.com/docs/aura/api/authentication/[Aura API credentials] to use Aura Graph Analytics.
+
+Optionally, you can create an example dataset in your Neo4j database using the following Cypher query.
+
+[source, cypher, copy=true]
+----
+CREATE
+  (a:User {name: 'Alice'}),
+  (b:User {name: 'Bob'}),
+  (c:User {name: 'Charlie'}),
+  (d:User {name: 'Dan'}),
+  (e:User {name: 'Eve'}),
+  (f:User {name: 'Frank'}),
+  (a)-[:LINK]->(b),
+  (a)-[:LINK]->(c),
+  (c)-[:LINK]->(e),
+  (e)-[:LINK]->(d),
+  (e)-[:LINK]->(f)
+----
+
+== Graph Analytics with Cypher
+
+Add data to your AuraDB instance using the Cypher query from the _Requirements_ section.
+Then, create a remote in-memory graph within an Aura Graph Analytics session using a _remote projection_.
+
+[source, cypher, copy=true]
+----
+CYPHER runtime=parallel
+WITH gds.aura.api.credentials("<Aura API Client ID>", "<Aura API Client Secret>") AS credentials
+MATCH (n)
+OPTIONAL MATCH (n)-->(m)
+RETURN gds.graph.project('myGraph', n, m, {}, {memory: '4GB'})
+----
+
+Once the graph is created, you can run any graph algorithms.
```

Comment:
```suggestion
Once the graph is created, you can run all supported graph algorithms.
```
We currently do not support all algorithms the plugin supports. Not sure if we need to mention this here